pyproject.toml

```
@@ -2,16 +2,11 @@
 # See LICENSE file for licensing details.
 
 [tool.poetry]
-name = "mongodb-k8s"
-version = "0.0.1-dev.0"
-description = ""
-license = "Apache-2.0"
-authors = [
-    "Mykola Marzhan <mykola.marzhan@canonical.com>"
-]
-readme = "README.md"
-homepage = "https://charmhub.io/mongodb-k8s?channel=6/edge"
-repository = "https://github.com/canonical/mongodb-k8s-operator"
+package-mode = false
+requires-poetry = ">=2.0.0"
+
+
+
```

I suggest this so we don't lose the metadata about the project.

```suggestion
package-mode = false
requires-poetry = ">=2.0.0"


[project]
name = "mongodb-k8s"
version = "0.0.1-dev.0"
description = ""
authors = [
    {name = "Mykola Marzhan", email = "mykola.marzhan@canonical.com"}
]
readme = "README.md"
requires-python = ">=3.10"

[project.urls]
homepage = "https://charmhub.io/mongodb-k8s?channel=6/edge"
repository = "https://github.com/canonical/mongodb-k8s-operator"
```

---

This pyproject.yaml was created before poetry became the standard of Data Platform charms, now that poetry is the standard, these fields are no longer used. I'd rather keep with the scheme from the other repos, but if you feel strongly just let me know and I can add this in!

---

Oh no it's okay if it's the standard elsewhere, go for it and keep it as is